tests/python/unittest/test_operator.py

```diff
@@ -7604,6 +7612,161 @@ def _softmaxoutput_normalization(multi_output, use_ignore, normalization):
                     multi_output, use_ignore, normalization)
 
 
+@with_seed()
+def test_argmax():
+    ci_test = True       # Use False if you need to collect aggregated information about argmax performance
```

Comment:
Thanks for adding the script for performance test. But I feel this is probably not the best place to add performance test in unit test. Maybe can you separate the performance test logic and add it to https://github.com/apache/incubator-mxnet/blob/master/tests/nightly/test_large_array.py?

Reply:
Thanks for the idea! In fact, I myself did not like a lot of `if`'s in this code. I have separated performance and unit test, but I think it would be better to keep
```
def profiling_argmax():
```
in `tests/python/unittest/test_operator.py`

The truth is: I tried to add it to `tests/nightly/test_large_array.py`, but it did not work for me. For some reason Python has some problem with this modules first lines:
```
root@627d15c17420:/opt/mxnet# python
Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> import mxnet as mx
>>> from mxnet.test_utils import rand_ndarray, assert_almost_equal, rand_coord_2d
>>> from mxnet import gluon, nd
>>> from tests.python.unittest.common import with_seed
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'tests.python'
>>> 
```